app/src/main/java/com/pwhs/quickmem/presentation/app/profile/choose_picture/ChoosePictureViewModel.kt

```
@@ -0,0 +1,23 @@
+import com.pwhs.quickmem.domain.repository.AuthRepository
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+@HiltViewModel
+class ChoosePictureViewModel @Inject constructor(
+    private val authRepository: AuthRepository
+) : ViewModel(){
+    init {
+
+    }
```

detekt

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyInitBlock)